src/asm/asm.h

```diff
@@ -92,72 +88,74 @@ typedef	struct	s_asm
 
 extern t_op		g_op_tab[17];
 
-void			free_split(char **split);
-void			clean_up(t_asm *a);
-void			print_error(void);
+bool			is_direct_label(char *str);
+bool			is_indirect_label(char *str);
+bool			is_label(char *str);
+bool			is_valid_operand(char *str);
+bool			ft_isspace(int c);
```

Comment:
Would you be willing to patch the libft to add this?

Currently it's included as a macro.

Reply:
This can be moved to libft later.